### PR TITLE
Alter the tokenEndpointAllowReusePvtKeyJwt form submittion logic

### PR DESCRIPTION
**Pull request by mpmadhavig**
### Purpose
$subject

### CODE REVIEW

**Review by pavinduLakshan — APPROVED** [review]

File: .changeset/yellow-wombats-grin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```
Comment (pavinduLakshan) [inline .changeset/yellow-wombats-grin.md:2]: this should be admin.applications.v1, not console

Comment (mpmadhavig) [reply .changeset/yellow-wombats-grin.md]: Fixed with 6b02ab6bdf7323798ec96467cf93e3ac68dd4690

**Review by pavinduLakshan — APPROVED** [review]

**Comment by Achintha444** [comment]
fix eslint issues

**Comment by mpmadhavig** [comment]
> fix eslint issues

Will be fixed in a later PR.

**Pull request merged**